BrainPortal/app/views/data_providers/show_custom.html.erb

```diff
@@ -0,0 +1,245 @@
+
```

Comment:
Delete this file completely, it's unnecessary. Like I said, the codebase's original 'show' page fully supported UserKey DPs.

Reply:
did not notice. Well, obviously be adjusted with extra embedded code, and logic in templates is harder to maintain due to mix of markup and embedded code